README.md

```diff
@@ -19,7 +19,6 @@ $ pip install mpegdash
 ## Test
 
 ```bash
-$ python -m unittest discover
```

Comment:
I think remove the other line, since `python3` is also somewhat an ancient concept nowadays.